packages/filebrowser/src/listing.ts

```diff
@@ -1810,23 +1815,27 @@ export namespace DirListing {
       if (fileType) {
         // TODO: remove workaround if...else/code in else clause in v2.0.0
         // workaround for 1.0.x versions of Jlab pulling in 1.1.x versions of filebrowser
-        if (this._iconRegistry) {
+        if (fileType.iconRenderer) {
           // add icon as svg node. Can be styled using CSS
-          this._iconRegistry.icon({
-            name: fileType.iconClass ?? '',
+          fileType.iconRenderer.render(icon, {
             className: ITEM_ICON_CLASS,
-            title: fileType.iconLabel,
-            fallback: true,
             container: icon,
+            title: fileType.iconLabel,
             center: true,
             kind: 'listing'
           });
         } else {
+          // cleanup after react
+          ReactDOM.unmountComponentAtNode(icon);
```

Comment:
```suggestion
          fileType.unrender(icon);
```

Reply:
I dropped the use of `render`/`unrender` from `listings` (see #7700).